### DISCUSSION TITLE
Busco equipo para hackaton :)

### DISCUSSION THREAD

**Discussion by Alefuentes982** [Launch-X-Latam/MisionFrontEnd#503] [Team Posts]
Disponible por acá para formar parte de algún equipo para la hackaton!! dejo mi correo: lx05961@innovaccion.mx, mi nombre es Alejandro, soy de Chile

**Comment by RocioSanchez2022** [comment]
Hola Alejandro! Yo también busco equipo.. soy de Argentina. Te gustaría que hagamos team? 😄

**Comment by LuisAlbertoCP** [comment]
Les gustaría integrarme a su equipo?  Yo soy de México

**Comment by Alefuentes982** [comment]
> Hola Alejandro! Yo también busco equipo.. soy de Argentina. Te gustaría que hagamos team? 😄

Claro!! sería excelente!!

**Comment by EliasQuino** [comment]
A mi me agregan mi correo  lx08662@innovaccion.mx

**Comment by EYacila** [comment]
Hola 👋 , también estoy disponible para unirme a un team.
Dejo mi correo ***@***.*** <http://innovaccion.mx/>*

**Comment by Alefuentes982** [comment]
> Les gustaría integrarme a su equipo? Yo soy de México

por mi parte, no hay problema!! buenisima

**Comment by itsmmunoz** [comment]
yo tambien me uno lx05555@innovaccion.mx

**Comment by dnahuirod** [comment]
Hola,  yo también busco team.

**Comment by Alefuentes982** [comment]
> Hola Alejandro! Yo también busco equipo.. soy de Argentina. Te gustaría que hagamos team? 😄

comparteme tu usuario de teams para agregarte Rocio

**Comment by LuisAlbertoCP** [comment]
@Alefuentes982 entonces si se hace el equipo me avisas bro y gracias 
Mi usuario  es 
lx03646@innovaccion.mx

**Comment by Alefuentes982** [comment]
> Disponible por acá para formar parte de algún equipo para la hackaton!! dejo mi correo: lx05961@innovaccion.mx, mi nombre es Alejandro, soy de Chile

compartan sus usuarios de teams para agregarlos porfa

**Comment by TheValentin** [comment]
hola puedo entrar a su grupo de hackaton, mi correo es: lx06687@innovaccion.mx , me llamo andy valentin soy de Perú

**Comment by RocioSanchez2022** [comment]
> Hola Alejandro! Yo también busco equipo.. soy de Argentina. Te gustaría que hagamos team? 😄
> 
> comparteme tu usuario de teams para agregarte Rocio

> lx06596@innovacion.mx

**Comment by irischinos** [comment]
Hola Ale y Rocío, yo también busco equipo, soy de México, ¿me podría unir al team? , mi correo es lx01713@innovaccion.mx

**Comment by iseddev** [comment]
Que tal compañeros, si aún hay lugar, me pueden incluir en el equipo 👍
Mi usuario: ***@***.***
Saludos desde México!!!
